File: practices/time_day.py
# SC 7th Time of Day

import datetime

now = int(input("Insert the current time for you in military time: \n"))
now_is = datetime.datetime.now()

if now >= 1 and now < 11:
    print("Good morning!")
elif now >= 1200 and now < 1730:
    print("Good Afternoon!")
elif now >= 1731 and now < 23:
    print("Good evening")
else:
    print("Are you dumb? That isnt a time idiot?")

File: c_programming/practices/time_day.c
// SC 7th Time of Day C
#include <stdio.h>
#include <time.h>

int main(void){
    time_t now = time(NULL);
    int time;
    printf("What is the current time for you in MILITARY TIME:\n");
    scanf("%d", &time);
    
    printf("The current time is %d\n", time);
    if (time <= 12){
        printf("Good Morning!\n");
    } else if (time < 17){
        printf("Good afternoon!\n");
    } else {
        printf("Good evening!\n");
    }
    return 0;
}